coverage_badge/__main__.py

```diff
@@ -35,6 +36,17 @@ def get_total():
     return '{0:.0f}'.format(total)
 
 
+def get_from_html(index_html):  # new
+    """
+    Return the total from index.html.
+    """
+    with open(index_html, "rb") as f:
```

Comment:
I'd probably open the file in text mode (so `open(indx_html, 'r') as f`). Or is there a specific reason it's using binary mode?